### DISCUSSION TITLE
Support for Azure OpenAI API version 2024-02-15-preview

### DISCUSSION THREAD

**Discussion by udione** [microsoft/semantic-kernel#5386] [General]
Yesterday, we've received email that Azure will retire old API's (including 2023-12-01-preview ) on 2nd April. (https://learn.microsoft.com/en-us/azure/ai-services/openai/api-version-deprecation)
Latest Microsoft.SemanticKernel (1.5) uses Azure.AI.OpenAI (1.0.0-beta.13) , which does not have the support for 2024-02-15-preview api. There is 1.0.0-beta.14, that has the support for Feb api but explicitly including that version has not worked for us (not supported exception).
What are the plans to update Microsoft.SemanticKernel to use 2024-02-15-preview. According to Microsoft communication in 3 weeks time old apis will stop working.

**Comment by cgan76** [comment]
We're also concerned about this, please advise.

**Comment by stephentoub** [comment]
https://github.com/microsoft/semantic-kernel/pull/5377 was merged yesterday to update to 1.0.0-beta.14 of the Azure.AI.OpenAI library. It should be in place for the next SK publish to nuget.
cc: @markwallace-microsoft

**Reply by cgan76** [reply]
Thanks, do you know when the next publish to nuget will be?

**Reply by stephentoub** [reply]
It was done earlier this week.
